setup.py

```diff
@@ -3,117 +3,108 @@
+                              re.sub(r"^(%s\/|%s$)" % (
+                                  os.path.join(package_name, 'libexec'),
+                                  os.path.join(package_name, 'libexec')),
+                                     "", subdir))
+    # Configuration directory
+    elif subdir and 'etc' in subdir:
+        target = os.path.join('etc/alignak',
+                              re.sub(r"^(%s\/|%s$)" % (
+                                  os.path.join(package_name, 'etc'),
+                                  os.path.join(package_name, 'etc')),
+                                     "", subdir))
+
+    if target is None:
+        print("Ignoring directory: %s" % (subdir))
+        continue
+
+    package_files = []
+    for file in files:
+        # Ignore files which name starts with __
+        if file.startswith('__'):
+            continue
+
+        package_files.append(os.path.join(subdir, file))
+
+    if package_files:
+        data_files.append((target, package_files))
+
+for (target, origin) in data_files:
+    print("Target directory: %s:" % (target))
+    for file in origin:
+        print(" - %s" % (file))
+print("====================================================\n")
 
 setup(
+    # Package name and version
     name=manifest["__pkg_name__"],
     version=manifest["__version__"],
 
     # Metadata for PyPI
     author=manifest["__author__"],
     author_email=manifest["__author_email__"],
     keywords="alignak monitoring module " + manifest["__module_types__"],
-    url=manifest["__url__"],
+    url=manifest["__git_url__"],
     license=manifest["__license__"],
     description=manifest["__description__"],
     long_description=open('README.rst').read(),
 
-    classifiers = [
-        'Development Status :: 5 - Production/Stable',
-        'Environment :: Console',
-        'Intended Audience :: Developers',
-        'Intended Audience :: System Administrators',
-        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
-        'Natural Language :: English',
-        'Programming Language :: Python',
-        'Topic :: System :: Monitoring',
-        'Topic :: System :: Systems Administration'
-    ],
+    classifiers = manifest["__classifiers__"],
 
     # Unzip Egg
     zip_safe=False,
 
     # Package data
     packages=find_packages(),
     include_package_data=True,
-    package_data={
-        '': 'README.rst',
-        '': 'AUTHORS',
-        '': 'LICENSE',
-        '': [os.path.join(manifest["__pkg_name__"], '*')],
-    },
-
-    # Where to install which file ...
-    # All pack files are installed at the same place.
+    # package_data={
+    #     '': 'README.rst',
+    #     '': 'AUTHORS',
+    #     '': 'LICENSE',
+    #     '': [os.path.join(manifest["__pkg_name__"], '*')],
+    # },
+
+    # Where to install distributed files
     data_files = data_files,
 
     # Dependencies (if some) ...
-    install_requires=['alignak_setup'],
+    install_requires=['requests', 'cherrypy'],
```

Comment:
All the modules require alignak ... but they also have their own requirements, and it is mandatory to define here.